ContentView.swift

```diff
@@ -0,0 +1,308 @@
+//
+//  ContentView.swift
+//  MusicApp
+//
+//  Created by Dawid Nowacki on 03/02/2024.
+//
+
+import SwiftUI
+import CoreData
+import AVKit
+
+struct ContentView: View {
+    
+    @State var audioFileName = "01 #1 - WIELKOMIEJSKA BEZSENNOŚĆ"
+    
+    @State private var player: AVAudioPlayer?
+    @State private var tytul: [String] = ["01 #1 - WIELKOMIEJSKA BEZSENNOŚĆ", "02 1-800-OŚWIECENIE", "03 #2 - GRZĘZNĄĆ W CISZY", "04 Cichosza", "05 #3 - TUTTO PASSA", "06 Gelato", "07 TU RADIO MARMUR #1", "08 Może To Coś Zmieni_", "09 Pakiet Platinium", "10 #4 - NESESER", "11 Makarena Freestyle", "12 #5 - GŁOSÓWKI DO REDAKCJI", "13 Mix Sałat", "14 #6 - DOBRY JAK DOBRY", "15 Codziennie", "16 #7 - PRACA DOMOWA", "17 1000 Dni Freestyle", "18 TU RADIO MARMUR #2", "19 Nametag", "20 #8 - KĄCIK KONSPIRACYJNY", "21 Całe Lata", "22 #9 - SŁYSZY PAN_", "23 Niedziela", "24 #10 - JESTEM", "25 Main Stage Freestyle", "26 #11 - LUSTERKO WSTECZNE", "27 EINECEIWŚO 008-1"]
```

Comment:
1. title instead of tytul
2. I would move this array to another file and create struct like Song with properties like title, filename, and other metadata stuff
With trying to mock API with method func fetchSongs() etc